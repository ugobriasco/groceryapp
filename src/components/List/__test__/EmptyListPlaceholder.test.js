import React from 'react';
import renderer from 'react-test-renderer';

import {EmptyListPlaceholder, styles} from '../';

describe('EmptyListPlaceholder', () => {

	it('renders correctly', () => {
	const tree = renderer.create(<EmptyListPlaceholder />)

	});

	it ('styles an object', () => {
		expect(typeof styles).toBe('object');
	});

	describe('_opt prop', () => {
		it('should render the empty-list placeholder if passed', () => {
			const rendered = renderer.create(<EmptyListPlaceholder _opt={'empty-list'}/>);
			expect(rendered).toMatchSnapshot();	
		});

		it('should render the empty-filter placeholder if passed', () => {
			const rendered = renderer.create(<EmptyListPlaceholder _opt={'empty-filter'}/>);
			expect(rendered).toMatchSnapshot();	
		});

	});

});



