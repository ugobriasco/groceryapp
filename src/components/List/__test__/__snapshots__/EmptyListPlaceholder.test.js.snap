// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyListPlaceholder _opt prop should render the empty-filter placeholder if passed 1`] = `
<View
  style={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={undefined}
  >
    No item found! You may add a new one
  </Text>
</View>
`;

exports[`EmptyListPlaceholder _opt prop should render the empty-list placeholder if passed 1`] = `
<View
  style={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={undefined}
  >
    Your list is empty!
  </Text>
</View>
`;
